src/snowflake/cli/_plugins/nativeapp/exceptions.py

```diff
@@ -111,3 +111,15 @@ class NoEventTableForAccount(ClickException):
 
     def __init__(self):
         super().__init__(f"{self.__doc__}\n\n{self.INSTRUCTIONS}")
+
+
+class ObjectPropertyNotFoundError(ClickException):
+    def __init__(self, property_name: str, object_type: str, object_name: str):
+        super().__init__(
+            dedent(
+                f"""\
+                        Could not find the '{property_name}' attribute for {object_type} {object_name} in the output of SQL query:
+                        'describe {object_type} {object_name}'
+                        """
+            )
+        )
```

Comment:
What does exactly this error represents? What users should do if they see such an error?

Reply:
How different is it from `DoesNotExistOrUnauthorizedError`?

Reply:
Sorry, both this (DoesNotExistOrUnauthorizedError) and ShowSpecificObjectMultipleRowsError should not be ClickExceptions since they are caused by us and not the user, just changed them to RuntimeErrors, thanks for the catch!